geomstats/tests.py

```diff
@@ -0,0 +1,45 @@
+import geomstats.backend as gs
+import unittest
+import tensorflow as tf
+import os
+
+
+class DummySession():
+    def __enter__(self):
+        pass
+
+    def __exit__(self, a, b, c):
+        pass
+
+
+def tf_backend():
+    return os.environ['GEOMSTATS_BACKEND'] == 'tensorflow'
+
+
+test_class = unittest.TestCase
+if tf_backend():
+    test_class = tf.test.TestCase
+
+
+class TestCase(test_class):
+    def assertAllClose(self, a, b, rtol=1e-6, atol=1e-6):
```

Comment:
`def assert_allclose` to favor in python / numpy naming style (not tf's)?

Reply:
actually the pythin testing is the following:

https://docs.python.org/2/library/unittest.html#test-cases

Since this is a testcase I think we might want to keep it like this because this is a class method